pakyow-presenter/lib/pakyow/presenter/front_matter_parser.rb

```diff
@@ -0,0 +1,36 @@
+module Pakyow
+  module Presenter
+    # Module for parsing front matter from HTML
+    module FrontMatterParser
```

Comment:
Make this api private by adding this:

```ruby
# @api private
```

This will prevent it from showing up in our api docs, since we only intend it for use by the framework.